### PR TITLE
open 1.14.8

### PR DESCRIPTION
**Pull request by patricklodder**
Open the 1.14.8 version so that we can compose a bugfix release

### CODE REVIEW

**Comment by patricklodder** [comment]
Marked urgent because ideally this goes in before everything else - it's also trivial.

**Review by chromatic — APPROVED** [review]
ACK, all tests pass on x86-64 Linux and code is correct.

**Comment by chromatic** [comment]
I'll hold this for an hour for comments and then any maintainer can merge, per the urgent label.

**Pull request merged**
